target/linux/ath79/image/tiny-netgear.mk

```diff
@@ -11,7 +11,8 @@ endef
 
 define Device/netgear_wnr612-v2
   $(Device/netgear_ar7240)
-  DEVICE_TITLE := Netgear WNR612v2
+  DEVICE_VENDOR := Netgear
+  DEVICE_MODEL := WNR612v2
```

Comment:
Should this be splitted?

Reply:
Yes, it is split in the device name netgear_wnr612-v2

Reply:
Updated, however this causes the first change of a device title, introducing a space between the variant...

Reply:
Yes, but I would frame it "fixing an inconsistency of device names".
Despite, I'm amazed that it's only one. Must be because ath79 is relatively new.